tests/js/sentry-test/reactTestingLibrary.tsx

```diff
@@ -75,7 +86,7 @@ function makeAllTheProviders(options: ProviderOptions) {
       </TestRouteContext.Provider>
     );
 
-    const history = createMemoryHistory();
+    const history = options.history ?? createMemoryHistory();
```

Comment:
Is this optional? Seems like we always pass this in now from `render`?

Reply:
`render` always passes it, but this function is used directly in some tests so I added the default for those cases

Reply:
Why are we doing that 😮‍💨